From 9682d4068ba906285956f90cf5fc4df8e6701081 Mon Sep 17 00:00:00 2001
From: Matt Jolly <kangie@gentoo.org>
Date: Wed, 13 Nov 2024 17:43:12 +1000
Subject: [PATCH] reclient_package: add condition checkout_x64

--- a/DEPS
+++ b/DEPS
@@ -996,7 +996,7 @@ deps = {
         'version': Var('reclient_version'),
       }
     ],
-    'condition': 'non_git_source',
+    'condition': 'checkout_x64 and non_git_source',
     'dep_type': 'cipd',
   },
 
-- 
2.47.0
